stdpopsim/slim_engine.py

```diff
@@ -181,7 +183,7 @@
     n_saves = 1 + sim.getValue("n_saves");
     sim.setValue("n_saves", n_saves);
     dbg("save() "+n_saves);
-    sim.treeSeqOutput(trees_file);
+    sim.treeSeqOutput(trees_file, metadata=metadata);
```

Comment:
This tree sequence is only an intermediate, so probably this doesn't need the metadata. No harm though I guess.

Reply:
It seems less confusing just to always have it in?